service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -32,6 +32,12 @@ public boolean isCtscEnabled(String localAuthorityName) {
             false);
     }
 
+    public boolean isCtscReportEnabled() {
+        return ldClient.boolVariation("CTSC",
+            ldUserBuilder.custom("report", true).build(),
```

Comment:
@pd-dudkiewicz what do you think about this approach? What do you think about this additional report config? 
Is it because now of three LAs enabled that we need to pass this through? 
We might need to consider limiting the email only to toggled LA?

Reply:
i already discusses with with Piotr, also discusses feature toggles with Gail, decision is to not filter out anything